src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/AzureVmModels/AzureRmRecoveryServicesAzureVmItem.cs

```diff
@@ -20,7 +20,57 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
 {
+    /// <summary>
+    /// Represents IaaSVM Item Class
+    /// </summary>
     public class AzureRmRecoveryServicesIaasVmItem : AzureRmRecoveryServicesItemBase
```

Comment:
I will check with PM for this. I think we should not use Azure keyword twice in the same string.